frontend/README.md

```diff
@@ -49,34 +53,37 @@ This is the easiest way to start developing, but it does not support all apis du
 development.
 
 Run `npm run mock:api` to start a mock backend api server handler so it can
-serve basic api calls with mock data. 
+serve basic api calls with mock data.
 
 ### Proxy to a real cluster
 
 This requires you already have a real KFP cluster, you can proxy requests to it.
 
-Before you start, configure your `kubectl` to talk to your KFP cluster.
+1. [Install Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/) based on your use case and environment.
+1. Configure your `kubectl` to access your KFP cluster. (For GCP, follow [Access to GCP cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) guide).
```

Comment:
Configure your `kubectl` to access -> Configure  `kubectl` with access to

Reply:
Applied the change.